acceptance/tests/ssl/certificate_extensions.rb

```diff
@@ -5,6 +5,8 @@
 test_name "certificate extensions available as trusted data" do
   confine :except, :platform => /^cisco_/ # See PUP-5827
 
+  skip_test "Test requires at least one non-master agent" if hosts.length == 1
+
   tag 'audit:high',        # ca/cert core functionality
       'audit:integration',
       'server'             # Ruby implimentation is deprecated
```

Comment:
How do you feel about a drive-by typo fix here?  `implimentation` -> `implementation`

Reply:
I'll fix it